src/nvim/testdir/bench_re_freeze.in

```diff
@@ -0,0 +1,13 @@
+Test for Benchmarking RE engine
```

Comment:
> We really want to get rid of the testdir/ directory

Absolutely, @justinmk. But then I need some assistance here. This test is not part of the old test suite, in fact the patch introduced a new make target `benchmark` to run it. It's a benchmark that doesn't verify functionality, it just does an expensive search and prints how long the search took. It takes about 15s to run and must be inspected by a human.

So what to do with it? I did a quick search for Lua benchmarking frameworks and didn't find anything. I don't think we want to make it part of the legacy test suite, it would slow things down a lot without adding any value.

That's why I just added it in this pull request. I don't know how to handle this atm.